### PR TITLE
PIPE-81-genophase-bam-cat

### PR DESCRIPTION
**Pull request by ottojolanki**
Implements combining bam using concatenation instead of merging. Should be significantly more efficient than before.

### CODE REVIEW

File: genophase.wdl [hunk 1]
```diff
@@ -81,6 +84,45 @@ workflow genophase {
     }
 }
 
+task concatenate_bams {
+    input {
+        Array[File] bams
+        Int disk_size_gb = 500
```
Comment (ian-whaling) [inline genophase.wdl]: I think this should be increased significantly. At least to 6000 to match `hic.merge`

Comment (ian-whaling) [reply genophase.wdl]: Also could you perhaps make disk size an input? I anticipate having to change it with respect to how many bams are inputted.

Comment (ottojolanki) [reply genophase.wdl]: I'll increase the default and make the disk size an optional workflow level input.

**Review by ian-whaling — CHANGES_REQUESTED** [review]
I think when you call concatenate_bams you need to call with the new input disk_size_gb = concatenate_bams_disk_size_gb

**Review by ian-whaling — APPROVED** [review]

**Pull request merged**